### ISSUE TITLE
Outlook 365 -- Composing -- When selecting text by word using shift + control + cursor keys, NVDA fails to read every other word

### ISSUE THREAD

**Issue by Luke-Robinett** [nvaccess/nvda#11918]
### Steps to reproduce:
1. Create a new message in Outlook 365
2. Type some text in the body of the message
3. Move the cursor to the start of the text
4. Hold down control and shift and use the cursor keys to select text by word

### Actual behavior:
NVDA only announces every other word. For example, performing the steps above for the sentence "Hello ladies and gentlemen" would yield the following output from left to right while selecting text by word: "hello," silence, "and," silence, "gentlemen."

### Expected behavior:
NVDA should read every word in the selection.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2020.3

#### Windows version:
Windows 10 Home, Windows 10 Pro (happens on both)

#### Name and version of other software in use when reproducing the issue:
Outlook 365

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes

**Comment by Qchristensen** [comment]
Which synthesizer are you using?

And which version of Office (I know 365, but Microsoft roll out different builds to different groups of people, so the fact I can't reproduce it won't mean as much if we're using different versions).  Also does it happen if you simply move by word without selecting?

Also, could you get a copy of your NVDA log, ideally at debug level to share here:

First of all, your NVDA key is either INSERT or CAPS LOCK, depending on how you have NVDA setup.  So, to set your log level:
1) Press NVDA+control+g to open the general settings
2) Press TAB until the focus is on 'Log level'
3) Press DOWN ARROW to get to 'Debug'
4) Press ENTER to close settings
5) Press NVDA+control+c to save settings.

To restart NVDA with add-ons disabled:
1) Press NVDA+Q
2) Down arrow to 'Restart with add-ons disabled'
3) Press ENTER

Next, recreate the issue - try selecting by Word in Outlook.

To get NVDA's log after that, there are several ways:

If NVDA is still running and usable:
1) Press NVDA+F1 to open the log viewer
2) Press CONTROL+A to select all.
3) Press CONTROL+C to copy.
4) Open your email and start a message to info@nvaccess.org, type a little about what you have done and what has happened in the body of the message, then leave a space and:
5) Press CONTROL+V to paste the copied log.

Instead of using the log viewer, or if NVDA has stopped and you needed to restart it or the computer:
1) Press WINDOWS+R to open Windows' Run dialog
2) Type %temp% and press ENTER (that's the percent sign, the letter t e m p and another percent sign).  Windows Explorer should open to the temporary folder.
3) Press TAB to move to the file list
4) Press N and move down to find up to three files:  nvda.log (the log file for the current or most recent NVDA session), nvda-old.log (the log from the previous session) and nvda-crash.dmp (a crash dump with more information created if NVDA itself crashes).

**Comment by Adriani90** [comment]
I cannot reproduce this issue with NVDA 2023.2 and Outlook 365, I am not using UIA for Word in advanced settings, it is set to "only when necessary".

I am closing this as abandoned since we didn't get any further updates from the initial author.
